.ado/templates/e2e-testing-uwp.yml

```diff
@@ -57,6 +57,12 @@ steps:
     condition: succeeded()
     displayName: 'Create success build variable'
 
+  - script: |
+      yarn appium driver install --source=npm appium-windows-driver
```

Comment:
This one needs the extra "-source=npm" and others don't?

Reply:
(same for win32)

Reply:
Yeah, because the windows driver is not in maintained by appium (though it kind of seems like it is), so it's not in their list of short names they recognize.